src/engine/engineaux.cpp

```diff
@@ -29,7 +29,6 @@ EngineAux::EngineAux(const QString& group, EffectsManager* pEffectsManager)
     // Default passthrough to enabled on the master and disabled on PFL. User
```

Comment:
there is nothing wrong with plf. I have just removed setPFL(false); because it is default.
Will improve the comment a bit.